sw/otbn/crypto/p384_isoncurve.s

```diff
@@ -145,10 +145,10 @@ p384_isoncurve:
  * This routine raises a software error and halts operation if the curve point
  * is invalid.
  *
- * @param[in]  dmem[12]: dptr_r, pointer to dmem location where right
- *                               side result r will be stored
- * @param[in]  dmem[16]: dptr_s, pointer to dmem location where left side
- *                               result s will be stored
+ * @param[in]  dmem[12]: dptr_rhs, pointer to dmem location where right
```

Comment:
Nit: I'd consider saying "right hand side" to match the "h" more obviously. Similarly for LHS.